openapi.yaml

```diff
@@ -0,0 +1,220 @@
+# this API specification is draft and subject to change.
+openapi: "3.0.2"
+info:
+  title: CVE Services API
+  description: The CVE services API which supports automation tooling for the CVE Project.
+  version: 0.1.0
+  contact:
+    "name": "API Support"
+    "url": http://cve.org/api
+    "email": api@mitre.org
+  license:
+    name: Creative Commons Zero v1.0 Universal
+    url: https://creativecommons.org/publicdomain/zero/1.0/legalcode
+  termsOfService: http://cve.org/api/terms
+# The following is notional
+servers:
+  - url: https://api-dev.cve.org/v1
+    description: Development and testing server
+  - url: https://api.cve.org/v1
+    description: Production server
+components:
+  parameters:
+    cve-id-path:
+      name: cve-id
+      in: path
+      description: The CVE id for which the record is being submitted
+      required: true
+      schema:
+        # CVE ID format
+        type: string
+        pattern: 'CVE-\d{4}-\d+'
+    api-entity-header:
+      name: X-API-ENTITY
+      description: >
+        The entity (e.g., CNA, ADP) that is authenticated or requesting authentication. \[TODO:: Is there a pattern for the format?\]
+      in: header
+      required: true
+      schema:
+        type: string
+    api-secret-header:
+      name: X-API-SECRET
+      description: >
+        The entity's (e.g., CNA, ADP) secret. \[TODO:: Is there a pattern for the format?\]
+      in: header
+      required: true
+      schema:
+        type: string
+paths:
+  # CVE submission and retrieval service
+  /cve/{cve-id}:
+    parameters:
+      - "$ref": "#/components/parameters/cve-id-path"
+    get:
+      summary: Retrieves a CVE record by id
+      operationId: getCVERecordById
+      responses:
+        200:
+          description: >
+            The requested CVE record associated with the CVE ID is returned.
+          content:
+            application/json:
+              schema:
+                type: object
+                # add schema for CVE 5.0 format
```

Comment:
Not sure where to find the current development JSON schemas. I can point remotely to it if there is a persistent URL to use across revisions to get the latest.